### ISSUE TITLE
Print error when room is not found

### ISSUE THREAD

**Issue by IanDeHaan** [Sooraj-beep/sherp2.0#17]

**Comment by Sooraj-beep** [comment]
The bot should respond to `?view f 23 ThisRoomDoesNotExist` with: 'The room you entered does not exist.'

**Comment by aarctan** [comment]
@DhanrajHira does https://github.com/Sooraj-beep/sherp2.0/pull/52 fix this?

**Comment by DhanrajHira** [comment]
@aarctan No it does not. I need to understand the Schedubuddy API better to make that work, Currently if the room does not exist, the API returns status 200 with some data that errors out the `draw_schedule` function.

**Comment by DhanrajHira** [comment]
@aarctan I would prefer if #52 is not blocked on this. The PR is already huge and I don't want more stuff to get tagged on. After the refactor introduced in that PR it will be relatively simple to add refinements so I'll prefer if this is deferred.
